codespell_lib/data/dictionary.txt

```diff
@@ -2102,7 +2139,9 @@ classe->class, classes,
 classess->classes
 classs->class
 classses->classes
+clcoksource->clocksource
 clcosed->closed
+cleancache->cleancacne
```

Comment:
This looks like it should be the other way around

Reply:
Fixed. Thanks.